lib/screens/home/ball_shooter/local_widget/customer_app_bar.dart

```diff
@@ -5,24 +5,24 @@ class CustomerAppBar extends StatelessWidget implements PreferredSizeWidget {
   final List<Widget>? action;
```

Comment:
I just noticed. `Customer` app bar doesn't make any sense since we don't have customers.